resources/wiki-overrides.properties

```diff
@@ -80,3 +80,5 @@ handlebars=https://github.com/jenkinsci/js-libs/blob/master/handlebars/README.md
 jquery-detached=https://github.com/jenkinsci/js-libs/blob/master/jquery-detached/README.md
 momentjs=https://github.com/jenkinsci/js-libs/blob/master/momentjs/README.md
 numeraljs=https://github.com/jenkinsci/js-libs/blob/master/numeraljs/README.md
+drecycler=https://github.com/jenkinsci/diskcheck-plugin/blob/master/README.MD
```

Comment:
https://github.com/jenkinsci/diskcheck-plugin/commit/4361fff9713ef5e8079f36ca90663f599f6bd10f

We should just suspend distribution instead and advertise as deprecated, there's never going to be a new release of `drecycler`.